src/dlite-codegen.c

```diff
@@ -13,6 +13,7 @@
 #include "dlite-macros.h"
 #include "dlite-codegen.h"
 
+#define GLOBALS_ID "dlite-codegen-globals-id"
```

Comment:
The GLOBALS_ID are defined in multiple files with different values and get_globals have several reimplementations as well. Consider refactoring.

Reply:
Yes, but all get_globals() are local to the file they are defined in and return different structures. Maybe it would be less confusing if we rename them according to the file their defined in. The same may be done with the GLOBALS_ID.